### ISSUE TITLE
rubocop_todo references cop configuration but could be more useful

### ISSUE THREAD

**Issue by maxjacobson** [bbatsov/rubocop#2322]
An example excerpt from my .rubocop_todo.yml

```yaml
# Offense count: 151                                                                
# Cop supports --auto-correct.                                                      
# Configuration parameters: EnforcedStyle, SupportedStyles.                         
Style/MultilineOperationIndentation:                                                
  Enabled: false                                                                    
```

The "Configuration parameters" line could be more useful, I think. I don't *really* care that it has a `SupportedStyles` configuration. However, I'm *very* interested that it has an `EnforcedStyle` parameter, but I need to hunt through the rubocop source code to determine what the valid options are to choose from. What do you think about including those options here?

**Comment by jonas054** [comment]
Sounds like a good idea. We can't go by name because these parameters are not always called `SupportedStyles`, but if we list the elements for all parameters whose type is `Array`, I think it will be an improvement.

```yaml
# Offense count: 151
# Cop supports --auto-correct.
# Configuration parameters: EnforcedStyle, SupportedStyles.
# SupportedStyles: aligned, indented
Style/MultilineOperationIndentation:
  Enabled: false
```

**Comment by maxjacobson** [comment]
That sounds great to me

**Comment by alexdowad** [comment]
Just [opened a PR](https://github.com/bbatsov/rubocop/pull/2522) which implements @jonas054's suggestion.

**Comment by maxjacobson** [comment]
:pray: sweet!
